tests/playwright/resources/test-kube-config

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
```

Comment:
Please add copyright.

Reply:
is it really necessary? it is not present in the other config files in resources, so either it is missing there as well or it is not needed...

Reply:
Yes, the others are missing it ;)